CMakeLists.txt

```diff
@@ -115,11 +115,17 @@ target_link_libraries(ed ed_core ed_io ed_server)
 #                                               PLUGINS
 # ------------------------------------------------------------------------------------------------
 
+find_package(tf2 REQUIRED)
+find_package(tf2_geometry_msgs REQUIRED)
+find_package(tf2_ros REQUIRED)
 add_library(ed_tf_publisher_plugin plugins/tf_publisher_plugin.cpp)
+target_include_directories(ed_tf_publisher_plugin PRIVATE ${tf2_geometry_msgs_INCLUDE_DIRS})
```

Comment:
How did this work before?

Reply:
I think because of dependencies of dependencies. But to my idea, this is the correct solution.

Reply:
I don't feel like going deeper into this ;-)